azure-pipelines-template.yml

```diff
@@ -61,6 +70,12 @@ jobs:
     - script: |
         rustup target add wasm32-unknown-unknown
       displayName: Add wasm32-unknown-unknown target
+  - ${{ if eq(variables['FEATURES'], 'winjack') }}
+    # Jack on Windows only
+    - powershell: |
+        Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+        choco install jack
```

Comment:
choco was the most straightforward way to do this since JACK only provides a graphical installer on windows